### ISSUE TITLE
Build raw-data from scratch?

### ISSUE THREAD

**Issue by jpressler** [DeFi-PortfolioManagement/defi-portfolio#20]
Hey,

First of all: Appreciating your work. Thanks for all your efforts.

My Question/RFE: 
Is there a way to build up the raw data from scratch?

Problem(s):
- When I switched my Wallet to another wallet.dat file still the old data is seen in the raw-data after updating, so there is a mix between the two wallets.
- Also, after rebuilding the DEFI-Blockchain from scratch with a snapshot, the old raw data ist still seen after updating.
- I also tried to unstinatll the defi-portfolio app and deleting the ~Users/AppData/Local/defi-portfolio Directory, and reinstall

I once deinstalled defi-app, defi-portfolio, cleaned-up the directory as above, deleted the blockchain (excl. /wallets) and reinstalled everything. Then it is working and the raw data got rebuilt from scratch.

Is there any other way to do this?
From which files/sources is the raw data built from and saved?

thx, best regards, Jochen

**Comment by DeFi-PortfolioManagement** [comment]
Hi Jochen,

sorry for the late reply.
You can try deleting the local transaction data in the folder
C:\Users\"Username"\AppData\Roaming\defi-portfolio\transactionsData.portfolio.
Then click on update data again. The data should then be downloaded again.

If that doesn't work, just contact us again :)
Best regards
Arthur

Am Fr., 5. März 2021 um 14:04 Uhr schrieb jpressler <
notifications@github.com>:

**Comment by jpressler** [comment]
Hi Arthur,
Thanks for your reply.

I tried what you have written and also deleted the whole blockchain, deinstall the your portfolio app and the wallet app and built up everything from scratch.

In both cases the raw data shows sent and receive amounts from the starting point till 20.Feb.2021 and no reward or commisions.

Beginning with Block 677967 on 01.March.2021 there are also reward and commision amount seen. strange ... Maybe it has something to do with the fork on 1st of march? Or maybe it is the time I switched to the new 2.3.0 wallet app with encryption (don't know exactly when that was but i think it was prior to 1st of march).

Luckily I saved the raw data up to 28.Feb.2021, event though this day may not reflect all comissions and rewards, but Jan/Feb are saved so far and I can continue with 01.March.2021. But I would like to help you find out, why the old data is not seen (as far as i can :-)).

best regards, Jochen

btw: I think you are from austria or germany? I'm from austria. we can catch up directly via eMail also in german joepre2102@gmail.com
